### DISCUSSION TITLE
Allow CP rebrand assets to be Asset element selection fields

### DISCUSSION THREAD

**Discussion by timkelty** [craftcms/cms#8517] [Ideas]
This would make migrating between environments nicer.

Especially if you're storing things on a S3 or whatever, it's a pain to remember you have to copy `storage/rebrand`, when everything else in there is temporal/disposable.

OR, and easier solution to implement might just be allowing the rebrand path to be configurable via config or constant.

**Comment by AugustMiller** [comment]
Finally realized why our `rebrand` changes wouldn't take (on Heroku)—we were setting the `CRAFT_STORAGE_PATH` to `/tmp`, which meant the `storage/rebrand` structure wasn't accessible.

I s'pose we could provide some method for copying those files at build time?

Pretty small beans, but it would be nice to have a solve or escape hatch in `craft\services\Path::getRebrandPath()`! 💞
